Dockerfile

```diff
@@ -11,4 +11,5 @@ COPY package*.json ./
 RUN npm install --production
 
 ENV NODE_ENV=production
+ENV TICKET_INVITE_KEY=$ticket_invite_key
```

Comment:
Should the build system be aware of the invite key? If not, it's enough to grab the variable from the environment at runtime.

```shell-script
$ gcloud run services update --update-env-vars=TICKET_INVITE_KEY=XXX SERVICE_NAME
```

Environment variables configured for a service persist between deployments unless explicitly cleared or updated.

Reply:
It has to exist when running `node` inside the docker image. If `--update-env-vars` on cloud run does that, it works for me!

Reply:
Yes, all configured environment variables are passed down to the container at runtime.

Reply:
Where should this be, then? Is it just setting variables on the cloud run instance, or do we need to explicitly run the command in say the cloudbuild file?

In other words: Can I just remove all docker/cloudrun environment variable management and just assume it exist in runtime?

Reply:
Yes.